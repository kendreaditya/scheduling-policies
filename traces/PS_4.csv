1,0,1
2,0,1
3,6,2
4,6,2
5,6,3
6,12,3
7,12,3
8,12,3
9,12,4
10,100,0
11,2000,3
12,2001,2
13,2002,1
14,2003,0
15,2003,1
16,2003,0
17,2004,0
18,2005,0
19,2005,3
20,2005,0
21,2006,0
22,2007,0
23,2008,0
24,2009,0
25,2010,0
26,2011,0
