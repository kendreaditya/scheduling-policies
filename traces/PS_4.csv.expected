1,2
2,2
3,12
4,12
5,17
6,25
7,25
8,25
9,26
10,100
11,2008
12,2008
13,2005
14,2003
15,2005
16,2003
17,2004
18,2005
19,2010
20,2005
21,2006
22,2007
23,2008
24,2009
25,2010
26,2011
